app/controllers/posts_controller.rb

```diff
@@ -5,8 +5,7 @@ class PostsController <  ApplicationController
 
   def index
     context = model.active.of_active_members
-
-    if current_organization.present?
+    if current_organization.present? && current_member.active?
```

Comment:
mmm 🤔  not sure about this one, posts index/show (read access) pages are public, so maybe we can avoid this check?

Reply:
With this check you show all offers imitating the behaviour you would have if you were not logged in. Without the check, you only see the current timebank's postings.

Reply:
mmm ok I understand your change, but still not sure how we should handle this, maybe we should discuss it with @sseerrggii too. Maybe we can just hide the "navbar", if you are not active in this organization and set current_organization to nil. So, we can save this logic to be spread around the code.

Reply:
Yes, we have to try to limit the logic at the beginning of the session, avoiding to complicate this functionality further.

To my understanding there are two possible cases
1. The user is an active user of some organizations: When entering, access the first organization in which he is an active member. (In the menu that allows you to change the organization, you will also have to go through this logic before changing to a new organization, also don't show on the menu the organizations where are you an inactive member)
  
2. The user is not an active member of any organization, I think we can prevent him from logging in and show him a public page with the information that he is not an active member of any organization?

Reply:
then I need to change the current login behaviour and the menu to change the organisation, right?

Reply:
No need @franpb14 , it was only a proposal, I saw the behaviour https://github.com/coopdevs/timeoverflow/pull/608#issuecomment-805019448 and it's ok for me :+1:  I think we need only to avoid assign current organization where you are an inactive member, what do you think?